mr_proper/main.py

```diff
@@ -7,29 +8,42 @@
 
 def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser()
-    parser.add_argument('filename', type=str)
+    parser.add_argument('objectname', type=str)
```

Comment:
This look like too generic name. How about `file_or_directory`?

Reply:
Done!